### PR TITLE
[pt] updated enum lesson

### PR DESCRIPTION
**Pull request by alessandraburckhalter**
Updated lesson: Enum #2396.

Updated the version of the lesson from 1.5.0 to 1.6.1 in Portuguese.

### CODE REVIEW

**Comment by doomspork** [comment]
Welcome and thank you @alessandraburckhalter!

File: pt/lessons/basics/enum.md [hunk 1]
```diff
@@ -207,3 +207,62 @@ iex> Enum.uniq([1, 2, 3, 2, 1, 1, 1, 1, 1])
 iex> Enum.uniq_by([%{x: 1, y: 1}, %{x: 2, y: 1}, %{x: 3, y: 3}], fn coord -> coord.y end)
```
Comment (antedeguemon) [inline pt/lessons/basics/enum.md:207]: Just a small change: the version in this markdown header needs to be updated to `1.6.1`.

**Review by antedeguemon — CHANGES_REQUESTED** [review]

File: pt/lessons/basics/enum.md (hunk 1 above)
Comment (alessandraburckhalter) [reply pt/lessons/basics/enum.md:207]: Done! Thank you.

**Review by antedeguemon — APPROVED** [review]

**Review by RafaelCamarda — APPROVED** [review]

**Review by wevtimoteo — APPROVED** [review]

File: pt/lessons/basics/enum.md [hunk 2]
```diff
@@ -207,3 +207,62 @@ iex> Enum.uniq([1, 2, 3, 2, 1, 1, 1, 1, 1])
...
+
+### Enum usando o operador Capture (&)
+
+Muitas funções dentro do módulo Enum no Elixir usam funções anônimas como um argumento para trabalhar com cada iterável do enumerável que é passado.
+
+Essas funções anônimas são frequentemente escritas de forma abreviada usando o operador Capture (&).
+
+Aqui estão alguns exemplos que mostram como o operador capture pode ser implementado com o módulo Enum. Cada versão é funcionalmente equivalente.
+
+### Usando o operador capture com uma função anônima
```
Comment (thiamsantos) [inline pt/lessons/basics/enum.md:219]:
```diff
-###
+####
```

@alessandraburckhalter Here we have a h4 in the original text.

File: pt/lessons/basics/enum.md [hunk 3]
```diff
@@ -207,3 +207,62 @@ iex> Enum.uniq([1, 2, 3, 2, 1, 1, 1, 1, 1])
...
+
+Isso pode ser refatorado para atribuir a função anônima anterior com o operador Capture a uma variável e chamada da função `Enum.map/2`.
+
+```elixir
+iex> plus_three = &(&1 + 3)
+iex> Enum.map([1,2,3], plus_three)
+[4, 5, 6]
+```
+
+### Usando o operador capture com uma função nomeada
```
Comment (thiamsantos) [inline pt/lessons/basics/enum.md:243]:
```diff
-### Usando o operador capture com uma função nomeada
+#### Usando o operador capture com uma função nomeada
```

@alessandraburckhalter Here we have a h4 in the original text.

**Comment by doomspork** [comment]
@thiamsantos what do we need to do here to merge?

**Comment by thiamsantos** [comment]
@doomspork I think we go ahead and merge it, I will open a different PR to fix the issues that raised here.

**Review by thiamsantos — APPROVED** [review]

**Pull request merged**